src/redux/configureStore.js

```diff
@@ -0,0 +1,8 @@
+import { combineReducers } from 'react-redux';
+import reducer from './books/books';
+import categories from './categories/categories';
+
+export default combineReducers({
+  reducer,
+  categories,
+});
```

Comment:
- [ ] It better to save combined reducers in store variable, and export it. 
- [ ] You are combining reducers but not creating a store, 
- [ ] I noticed that you called one reducer as 'reducer' , I strongly recommend renaming it to bookReducer, this way your code can be maintainable and readable. 

Here is an example of your configStore: 
```
import { combineReducers, createStore } from 'react-redux';
const rootReducer = combineReducers({
  categories: categoryReducer,
  books: booksReducer,
});

const store = createStore(rootReducer);

export default store;
```